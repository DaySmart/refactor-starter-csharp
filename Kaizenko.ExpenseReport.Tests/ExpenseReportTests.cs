using ApprovalTests.Reporters;
using NUnit.Framework;
using System;
using System.IO;

namespace Kaizenko.ExpenseReport.Tests
{
    public class ExpenseReportTests
    {
        [Test]
        [UseReporter(typeof(DiffReporter))]
        public void PrintReport_WhenNoExpensesProvided_ExpectAnEmptyReport()
        {
            using (StringWriter sw = new StringWriter())
            {
                Console.SetOut(sw);
                ExpenseReport expenseReport = new ExpenseReport();
                expenseReport.PrintReport(new System.Collections.Generic.List<Expense>());
                ApprovalTests.Approvals.Verify(sw);
            }
        }
    }
}
