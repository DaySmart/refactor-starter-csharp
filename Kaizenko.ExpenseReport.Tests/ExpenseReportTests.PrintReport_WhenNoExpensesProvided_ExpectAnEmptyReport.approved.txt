Expenses 9/1/2022 10:32:59 AM
Meal expenses: 0
Total expenses: 0
